plugins/modules/aci_access_span_src_group_src.py

```diff
@@ -0,0 +1,442 @@
+  returned: failure or debug
+  type: int
+  sample: 200
+url:
+  description: The HTTP url used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: https://10.11.12.13/api/mo/uni/tn-production.json
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.cisco.aci.plugins.module_utils.aci import ACIModule, aci_argument_spec, aci_annotation_spec, aci_owner_spec
+from ansible_collections.cisco.aci.plugins.module_utils.constants import SPAN_DIRECTION_MAP
+
+
+def main():
+    argument_spec = aci_argument_spec()
+    argument_spec.update(aci_annotation_spec())
+    argument_spec.update(aci_owner_spec())
+    argument_spec.update(
+        source_group=dict(type="str", aliases=["src_group"]),  # Not required for querying all objects
+        description=dict(type="str", aliases=["descr"]),
+        source=dict(type="str", aliases=["name", "src"]),  # Not required for querying all objects
+        direction=dict(type="str", choices=list(SPAN_DIRECTION_MAP.keys())),
+        filter_group=dict(type="str"),
+        drop_packets=dict(type="bool"),
+        epg=dict(
+            type="dict",
+            options=dict(
+                epg=dict(type="str", required=True, aliases=["epg_name"]),
+                ap=dict(type="str", required=True, aliases=["ap_name", "app_profile", "app_profile_name"]),
+                tenant=dict(type="str", required=True, aliases=["tenant_name"]),
+            ),
+        ),
+        routed_outside=dict(
+            type="dict",
+            options=dict(
+                encap=dict(type="int", aliases=["vlan", "vlan_id", "encap_id"]),
+                l3out=dict(type="str", aliases=["l3out_name"]),
+                tenant=dict(type="str", aliases=["tenant_name"]),
+            ),
+        ),
+        state=dict(type="str", default="present", choices=["absent", "present", "query"]),
+        name_alias=dict(type="str"),
+    )
+
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        required_if=[
+            ["state", "absent", ["source_group", "source"]],
+            ["state", "present", ["source_group", "source"]],
+        ],
+        mutually_exclusive=[
+            ("epg", "routed_outside"),
+        ],
+    )
+
+    aci = ACIModule(module)
+
+    description = module.params.get("description")
+    source_group = module.params.get("source_group")
+    source = module.params.get("source")
+    direction = module.params.get("direction")
+    filter_group = module.params.get("filter_group")
+    drop_packets = module.params.get("drop_packets")
+    epg = module.params.get("epg")
+    routed_outside = module.params.get("routed_outside")
+    state = module.params.get("state")
+    name_alias = module.params.get("name_alias")
+
+    if filter_group and drop_packets:
+        module.fail_json(msg="It is not allowed to configure 'drop_packets: true' when 'filter_group' is configured on the source.")
```

Comment:
Just a suggestion -
"Setting 'drop_packets' to 'true' is not allowed when 'filter_group' is configured on the source."
 "Setting 'drop_packets' to 'true' is not allowed when 'epg' is configured on the source."
"Setting 'drop_packets' to 'true' is not allowed when 'routed_outside'  is configured on the source."